src/components/VaccineProgressBar/VaccineProgressBar.style.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+import { COLOR_MAP } from 'common/colors';
+import { materialSMBreakpoint } from 'assets/theme/sizes';
+
+export const ProgressBarContainer = styled.div`
+  width: 100%;
+  max-width: 350px;
+  margin: auto;
+
+  @media (min-width: ${materialSMBreakpoint}) {
+    max-width: 288px;
+    margin: 0;
+  }
+`;
+
+export const StyledSvg = styled.svg`
+  border: 1px solid ${COLOR_MAP.GREY_2};
```

Comment:
i think this is meant to be `COLOR_MAP.GREY_1`

Reply:
Leaving it GREY_2 to better match the mocks with the new design.